governance.md

```diff
@@ -129,6 +130,21 @@ committee has a charter and a chair, and will report to the steering
 committee periodically, and to the community as makes sense, given the
 charter.
 
+## User groups
+Some topics have long term relevance to large groups of Kubernetes users, but
+do not have clear deliverables or ownership of parts of the Kubernetes
+code base. As such they are neither good fits for SIGs or Working Groups.
+Examples of such a topic might be continuous delivery to Kubernetes.
+
+User groups function as a centralized resource to facilitate communication
+and discovery of information related to the topic of the user group. User
+groups should not undertake to produce any deliverable, instead they should
```

Comment:
I've thought in the past that it would be useful for such groups to convey requirements or friction for their use case to the relevant SIGs. That could be as simple as filing bugs or SIG presentations, but I could see cases where docs might be useful. These don't necessarily need to live within the project, though.

So, perhaps encourage engagement with SIGs for issues that don't rise to the level of requiring a WG.

Reply:
Added some text that tries to capture this.